src/Flex.php

```diff
@@ -130,7 +130,7 @@ public function configurePackage(PackageEvent $event): void
 
             $manifest = $recipe->getManifest();
             if (isset($manifest['post-install-output'])) {
-                $this->postInstallOutput = array_merge($this->postInstallOutput, $manifest['post-install-output'], ['']);
+                $this->postInstallOutput = array_merge($this->postInstallOutput, $this->options->expandTargetDir($manifest['post-install-output']), ['']);
```

Comment:
this is broken. `$manifest['post-install-output']` is an array of lines. You need to loop over the array